doc/manual/manual.tex

```diff
@@ -8907,6 +8910,50 @@ \subsubsection{The van Keken thermochemical composition benchmark}
 
 We can observe that when the two-layer viscosity model is added, there is only one apparent peak for each case. The first peaks of the 0.01 viscosity contrast tests appear earlier and are larger in magnitude than those of 0.1 viscosity contrast tests.  There are no secondary plumes and the whole system tends to reach stability after around 500 time steps.
 
+\subsubsection{Polydiapirism}
+\label{sec:benchmark-polydiapirism}
+
+\textit{This section was contributed by Cedric Thieulot.}
+
+Diapirs are a type of geologic intrusion in which a more mobile and ductily deformable material (e.g., salt)
+is emplaced into (brittle) overlying rocks. As salt domes are capable of trapping petroleum and 
+natural gas these structures have been extensively studied \cite{jahu17}.
+
+We consider in this experiment the three-layer viscous Rayleigh-Taylor instability proposed 
+by Weinberg and Schmeling \cite{wesc92} 
+and we focus in what follows on the case II of that publication.
+The domain is a 2D Cartesian box of size 2.24m$\times$1m. Gravity is Earth-like ($9.81\text{m}/\text{s}^2$).
+Boundary conditions are free-slip on the sides and top and no-slip at the bottom.
+All three layers are initially horizontal. The top layer (fluid 1) has a thickness of 
+0.75m, a viscosity $\eta_1=100$ Pa$\cdot$s and a density $\rho_1=100$kg/m$^3$. 
```

Comment:
One more of these:
```suggestion
0.75m, a viscosity $\eta_1=100$ Pa$\cdot$s and a density $\rho_1=100 \text{kg}/\text{m}^3$. 
```
There might be more in this section I'm missing right now. I believe I remember that just having `$^3$` leads to error messages in the programs that convert latex into HTML. @tjhei wants to use these to make the manual more easily accessible.